docusaurus.config.js

```diff
@@ -104,6 +104,11 @@ const config = {
             label: "Top DSA Interview Questions",
             position: "left",
           },
+          {
+            to: "Visualizing",
+            label: "Sorting Visualizer",
+            position: "left",
+          },
```

Comment:
delete lines `107` to `111`

Reply:
so how can i defined that new feature ? in navbar i've addedthis..

Reply:
I removed it.. please Review Again...

Reply:
i removed it but when should i render it ? should i render it in home page ?

Reply:
requested to guid me